eth-rpc-adapter/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:14-alpine
+
+ARG mandala_url=wss://node-6870830370282213376.rz.onfinality.io/ws?apikey=0f273197-e4d5-45e2-b23e-03b015cb7000
```

Comment:
No need to inject this at build time. We can configure environment variable at deployment time.